src/main/app/src/interfaces.ts

```diff
@@ -1,10 +1,35 @@
 export interface ILandscape {
-    name: string;
-    identifier: string;
-    description?: string;
-    contact?: string;
-    teams: string[];
-    groups?: string[];
-    items?: string[];
-    lastUpdate?: string;
+  name: string;
+  identifier: string;
+  description?: string;
+  teams?: object[]; // TODO: Create interface
+  items?: IItem[];
+  groups?: object[]; // TODO: Create interface
+  lastUpdate?: string;
+  overallState?: string;
+}
+
+export interface IItem {
+  color?: string;
+  contact?: string;
+  description?: string;
+  fill?: string;
+  fullyQualifiedIdentifier?: string;
+  group?: string;
+  height?: number;
+  icon?: string;
+  identifier: string;
+  interfaces?: Array<Object>;
+  labels?: Object;
+  lifecycle?: String;
+  links?: Object;
+  name?: String;
+  owner?: String;
+  providedBy?: Array<Object>;
```

Comment:
Pls check if this appears at all. ProvidedBy is a shortcut for configuring landscapes long before internal processing is done.

Reply:
i've added all attributes that got send within the response, idk if its used anywhere but it is send

Reply:
OK. Then we should #137 later.